llvm_util/llvm2alive.cpp

```diff
@@ -998,6 +1003,9 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
 
 namespace llvm_util {
 
+unsigned omit_array_size = -1;
+
```

Comment:
This is declared here rather than at ir/globals.h because constants at globals.h are determined after an Alive2 function is analyzed.
Also, moving this to globals.h introduces dependency to ir/globals.h , which did not exist before.